services/cd-service/pkg/argocd/v1alpha1/types.go

```diff
@@ -147,9 +147,11 @@ type SyncWindow struct {
 
 // ResourceIgnoreDifferences contains resource filter and list of json paths which should be ignored during comparison with live state.
 type ResourceIgnoreDifferences struct {
-	Group        string   `json:"group,omitempty" protobuf:"bytes,1,opt,name=group"`
-	Kind         string   `json:"kind" protobuf:"bytes,2,opt,name=kind"`
-	Name         string   `json:"name,omitempty" protobuf:"bytes,3,opt,name=name"`
-	Namespace    string   `json:"namespace,omitempty" protobuf:"bytes,4,opt,name=namespace"`
-	JSONPointers []string `json:"jsonPointers" protobuf:"bytes,5,opt,name=jsonPointers"`
+	Group                 string   `json:"group,omitempty" protobuf:"bytes,1,opt,name=group"`
+	Kind                  string   `json:"kind" protobuf:"bytes,2,opt,name=kind"`
+	Name                  string   `json:"name,omitempty" protobuf:"bytes,3,opt,name=name"`
+	Namespace             string   `json:"namespace,omitempty" protobuf:"bytes,4,opt,name=namespace"`
+	JSONPointers          []string `json:"jsonPointers,omitempty" protobuf:"bytes,5,opt,name=jsonPointers"`
+	JqPathExpressions     []string `json:"jqPathExpressions,omitempty" protobuf:"bytes,6,opt,name=jqPathExpressions"`
```

Comment:
Either in this or in a separate PR, add to the changelog something like
"Added support to ignore argo fields jsonpointers and jqpath"